data/ha/smb.conf

```diff
@@ -0,0 +1,38 @@
+[global]
+    netbios name = SLE15SP1
```

Comment:
Maybe a more generic name could be used here?

Reply:
Yes I forgot to change it, CTDBTEST is used now. Thx for pointing out this.